src/app/profiles/profile.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { ReduxModule } from '../redux/redux.module';
+
+import {
+    ProfileFormViewComponent,
+    ProfileFormComponent,
+} from './index';
+
+@NgModule({
+  imports: [
+    BrowserModule,
```

Comment:
I get a compiler error, because the `Async pipe` was not defined.   Importing the BrowserModule here solved the error.  I don't know why the compilation could not resolve the async pipe from the `BrowserModule` in the app module.